include/qml_ros2_plugin/service_client.hpp

```diff
@@ -33,12 +35,18 @@ class ServiceClient : public QObjectRos2
   Q_OBJECT
   //! True if the ServiceClient is connected to the Service and the Service is ready, false otherwise.
   Q_PROPERTY( bool ready READ isServiceReady NOTIFY serviceReadyChanged )
+
+  //! The quality of service settings for this service client's connection.
+  //! See QoS and the ROS2 Docs for more info:
+  //! https://docs.ros.org/en/rolling/Concepts/About-Quality-of-Service-Settings.html
+  Q_PROPERTY( qml_ros2_plugin::QoS qos READ qos )
```

Comment:
Tried to make this a read-only attribute, as I got the impression `ServiceClient` is not as 'dynamic' as `Publisher` and `Subscription` are (ie: no changing of QoS while object exists).

So there's no `qosChanged` signal nor a setter here.